src/backend/internal/controller/user/get_photographer_role.go

```diff
@@ -0,0 +1,57 @@
+package user
+
+import (
+	"net/http"
+
+	"github.com/Roongkun/software-eng-ii/internal/model"
+	"github.com/gin-gonic/gin"
+	"github.com/google/uuid"
+)
+
+func (r *Resolver) GetPhotographerRole(c *gin.Context) {
+	user, exists := c.Get("user")
+	if !exists {
+		c.JSON(http.StatusInternalServerError, gin.H{"status": "failed", "message": "Failed to retrieve user from context"})
```

Comment:
We have just adopted a new convention for returning errors. 

All the error message should be returned in the `error` field.
Please do change the field `message` to `error` for every early-return block.